NN_chatCmd.js

```diff
@@ -12,17 +13,28 @@ nerthus.chatCmd.run = function(ch)
         var callback = this.fetch_callback(cmd,ch)
         if(callback)
         {
-            log("["+ch.k+"] " + ch.n + " -> " + ch.t) //gdze kto co
-            return callback(ch)
+            log("["+ch.k+"] " + ch.n + " -> " + ch.t +
+            " (" + nerthus.chatCmd.current_map +")"); //gdzie kto co mapa
+            return callback(ch);
         }
     }
 }
 
 nerthus.chatCmd.fetch_cmd = function(ch)
 {
-    if(ch.t[0]=='*')
-        return RegExp(/^\*(\S+)/).exec(ch.t)
-    return null
+    if (nerthus.chatCmd.currentMap === '')
+        nerthus.chatCmd.currentMap = map.name.replace(/\s/g, '');
```

Comment:
Nie łatwiej używać do tego map.id? nie trzeba wtedy robić różnych dziwnych manewrów typu wywalanie spacji.

Reply:
entertown nie daje id mapy na którą się weszło tylko samą jej nazwę, dlatego tak to załatwiłem